translations/messages.fr.xlf

```diff
@@ -0,0 +1,3063 @@
+        <target>Bravo, vous avez récupéré 4 de vos pièces et capturé 2 pièces de l&apos;adversaire.
+        Le maximum possible étant 3 par captures.</target>
+      </trans-unit>
+      <trans-unit id="3200525134996933550" datatype="html">
+        <source>Failed, the best capture was taking 2 of your opponent&apos;s pieces</source>
+        <target>Raté, la capture optimale capture 2 pièces adverses.</target>
+      </trans-unit>
+      <trans-unit id="6859741057618390920" datatype="html">
+        <source>Capturing (3/3)</source>
+        <target>Capture (3/3)</target>
+      </trans-unit>
+      <trans-unit id="1697647848066799342" datatype="html">
+        <source>Here, you must capture at the beginning of your turn.
+        This is due to a move of your opponent at the previous turn
+        (even though this is a fictional game for pedagogical purpose).
+        After your capture, by performing the right move you can even capture two more of your opponent&apos;s pieces!
+        Keep it mind that the most useful in a capture is to take your opponent&apos;s pieces.</source>
+        <target>Ici, vous aurez une capture à faire au début de votre tour.
+        Elle a été provoquée par un mouvement de votre adversaire lors de son tour de jeu
+        (bien que ce plateau soit fictif à des fins pédagogiques).
+        En effectuant ensuite le bon mouvement, vous pourrez faire deux captures supplémentaires !
+        Gardez à l&apos;esprit que le plus utile d&apos;une capture, est de capturer les pièces adverses !</target>
+      </trans-unit>
+      <trans-unit id="1212338036122995731" datatype="html">
+        <source>Congratulations, you have gotten 12 of your pieces back and captured 2 of your opponent&apos;s pieces.
+        The maximum being 3 per capture.</source>
+        <target>Bravo, vous avez récupéré 12 de vos pièces et capturé 2 pièces de l&apos;adversaire.
+        Le maximum possible étant 3 par captures.</target>
+      </trans-unit>
+      <trans-unit id="4244295242962463153" datatype="html">
+        <source>Failed, the best capture takes 2 of your opponent&apos;s pieces.</source>
+        <target>Pièces du joueur <x id="INTERPOLATION" equiv-text="{{ player + 1 }}"/></target>
+      </trans-unit>
+      <trans-unit id="4172293183843503071" datatype="html">
+        <source>This move is a ko, you must play somewhere else before you can play in this intersection again.</source>
+        <target>Ce mouvement est un ko, vous devez jouer ailleurs avant de pouvoir rejouer dans cette case</target>
+      </trans-unit>
+      <trans-unit id="9197288705784594537" datatype="html">
+        <source>We are not in playing nor in passed phase, you must mark stone as dead or alive or accept current board.</source>
+        <target>Ce n'est pas une phase de jeu ou une phase où il faut passer, vous devez marquer les pierres comme mortes ou vivantes, ou bien accepter l'état actuel du plateau..</target>
```

Comment:
We are in counting phase* (pareil pour français)

Sinon oui, c'est pas de la review de traduction mais bon, en même temps on repasse sur tout le texte ça vaut le coup ?

Reply:
phase de comptage en FR ?
Aussi: comment est-ce qu'on accepte le plateau, en passant ? Si oui, c'est à intégrer dans l'explication je dirais

Reply:
Phase de comptage oui, la phase ou on compte (là il manque un espace, concentre toi qu'on gagne deux trois pongs :D)

On peut accepter en passant son tour, à mentionner en FR et EN, bien vu !

Sinon, double point :D ! This is ze kikulel langage :D !

Reply:
Voilà repassé correctement dessus j'espère cette fois ! J'suis aussi repassé sur l'EN qui sonnait très robotique :D